client/src/index.js

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
-import ReactDom from 'react-dom';
+import ReactDOM from 'react-dom';
+import App from './components/App.jsx';
 
-ReactDom.render(<div>Hello World</div>, document.getElementById('app'));
+
+ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
Would there be a future conflict if we all call our modules App?

Reply:
I'm either going to change the name to photo or hash it